### PR TITLE
Migration to package format 2

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_default_env_config/package.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roslaunch</build_depend>  
+  <build_depend>roslaunch</build_depend>
```
Comment (NadiaHG) [inline cob_default_env_config/package.xml]: roslaunch is only a build_depend or also an exec_depend? For other packages (like cob_bringup or cob_default_robot_config)  roslaunch is a depend (build+exec) , what is the difference?

**Comment by mgruhler** [comment]
LGTM

File: cob_default_env_config/package.xml (hunk 1 above)
Comment (fmessmer) [inline cob_default_env_config/package.xml]: made consistent

**Comment by NadiaHG** [comment]
LGTM

**Pull request merged**